go/parquet/internal/encoding/delta_byte_array_test.go

```diff
@@ -0,0 +1,31 @@
+package encoding
```

Comment:
You need to add the Apache license to the top of this file please. That will fix the failing CI